setup.cfg

```diff
@@ -83,9 +83,10 @@ coreg = earthpy
 zarr = zarr
     numcodecs
 ml = dask-ml>=2022.5.27
-    scikit-learn>=0.23.0,<=1.2.0
+    scikit-learn=1.2.0
```

Comment:
Is there any way we can make this `scikit-learn>=1.2.0`?

Reply:
Do you know what needs updated in `sklearn-xarray`? We are installing a forked version so we can modify it. 

I'm a bit hesitant to enforce such a hard dependency pin.

Reply:
At the moment I don't think we have much of an option. My plan is to take a look at this in about 2 weeks when I get done with classes. On a positive note, I think this only comes into play with the machine learning add-ons. So it should n't affect very many users for the moment.